modules/nw-metallb-bgppeer-cr.adoc

```diff
@@ -26,14 +26,21 @@ The fields for the BGP peer custom resource are described in the following table
 
 |`spec.myASN`
 |`integer`
-|Specifies the Autonomous System number for the local end of the BGP session.
+|Specifies the Autonomous System Number (ASN) for the local end of the BGP session.
 Specify the same value in all BGP peer custom resources that you add.
```

Comment:
```suggestion
In all BGP peer custom resources that you add, specify the same value.
```
Because the line just before this starts with "Specifies", consider revising this line to start differently.

Reply:
Thanks for the review @lahinson 
Makes sense